src/ArmTemplates/Common/Templates/Policy/PolicyTemplateProperties.cs

```diff
@@ -4,11 +4,14 @@
 //  </copyright>
 // --------------------------------------------------------------------------
 
+using Newtonsoft.Json;
+
 namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Policy
 {
     public class PolicyTemplateProperties
     {
-        public string Value { get; set; }
+        [JsonProperty("value")]
```

Comment:
Shouldn't this be "Value"?

Reply:
For me it is not very readable - you are working in code and you have a `Value` for PolicyProperties. What is value? maybe a content, maybe a name of policy - only god knows. That is why I decided to provide a readable name and make use of `JsonProperty` attribute :)

Also in code I saw some comments that are describing what **value** is, and I find it *only* confusing
```
// value is a content
template.Content = policyTemplateProperties.value;
```

Reply:
I'm talking about the JsonProperty string param. Does it need to be "Value" instead of "value" to be backwards compatible?

Reply:
No, it is for the serialization purposes.
It is `value` in the json:
![image](https://user-images.githubusercontent.com/31598696/157685678-839636a4-945d-4bc0-bd1b-05884cd71126.png)

And JsonProperty name is case sensitive, that is why I wrote `value`, not `Value`.